src/ccsr/src/config.rs

```diff
@@ -63,7 +64,13 @@ impl ClientConfig {
 
     /// Builds the [`Client`].
     pub fn build(self) -> Client {
-        let mut builder = reqwest::Client::builder();
+        let mut builder = match ore::http::reqwest_client_builder() {
+            Ok(builder) => builder,
+            Err(e) => {
+                warn!("Unable to construct a reqwest client that will obey the http_proxy settings: {}", e);
```

Comment:
We should just propagate the error here, ultimately bubbling it up to whoever creates the ccsr client. Warning logs like this will never get looked at, and silently using an HTTP connector that doesn't use the proxy is not a great fallback IMO.

Reply:
Done.